src/test/java/org/zalando/nakadi/security/AuthenticationTest.java

```diff
@@ -76,7 +76,7 @@
 import static org.zalando.nakadi.utils.TestUtils.randomUUID;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = Application.class, webEnvironment = WebEnvironment.RANDOM_PORT)
+@SpringBootTest(classes = Application.class, webEnvironment = WebEnvironment.RANDOM_PORT, value = {"management.port=0"})
```

Comment:
@antban what's this change for?

Reply:
In case if unit tests are running in parallel, they are trying to use the same management port, defined in configuration. I just specified, that the port should be random (the same as WebEnvironment.RANDOM_PORT, but for management port)